src/Orckestra.Media.AutoImageResizing/Helpers/AutoImageResizingHelper.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Composite.Core;
+using Composite.Core.Routing;
+using Composite.Core.WebClient;
+using Composite.Core.WebClient.Media;
+using Composite.Data;
+using Composite.Data.Types;
+
+namespace Orckestra.Media.AutoImageResizing.Helpers
+{
+    public static class AutoImageResizingHelper
+    {
+        public static bool IsLocalC1MediaWithoutResizingOptions(string relativeUrl)
+        {
+            var mediaUrlUnprocessedInternalPrefix = "~/media(";
+            var mediaUrlInternalPrefix = UrlUtils.PublicRootPath + "/media(";
+            var mediaUrlPublicPrefix = UrlUtils.PublicRootPath + "/media/";
+
+            var parsedUrl = new UrlBuilder(relativeUrl);
+            var queryParameters = parsedUrl.GetQueryParameters();
+            var isEmptyResizingQueryParameters = ResizingOptions.Parse(queryParameters).IsEmpty;
+
+            return (relativeUrl.Contains(mediaUrlUnprocessedInternalPrefix)
+                    || relativeUrl.Contains(mediaUrlInternalPrefix)
+                    || relativeUrl.Contains(mediaUrlPublicPrefix))
+                   && isEmptyResizingQueryParameters;
```

Comment:
The code will work a bit faster, if you don't parse the query string parameters, unless you already know it is a media URL:

```
private static readonly string MediaUrlUnprocessedInternalPrefix;
private static readonly string MediaUrlInternalPrefix = UrlUtils.PublicRootPath + "/media(";
private static readonly string MediaUrlPublicPrefix = UrlUtils.PublicRootPath + "/media/";

...

if (!relativeUrl.StartsWith(MediaUrlUnprocessedInternalPrefix)
	&& !relativeUrl.StartsWith(MediaUrlInternalPrefix)
	&& !relativeUrl.StartsWith(MediaUrlPublicPrefix)) 
{
	return false;
}

var parsedUrl = new UrlBuilder(relativeUrl);
var queryParameters = parsedUrl.GetQueryParameters();

return ResizingOptions.Parse(queryParameters).IsEmpty;
```